plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1539,6 +1539,22 @@ public void testTableWithTrailingSlashLocation(boolean partitioned)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAnonymousRowTypeFailure()
+    {
+        String expectedMessage = "Anonymous row type is not supported in Delta Lake connector";
+        String tableName = "test_anonymous_row_type_" + randomNameSuffix();
+
+        String createTable = "CREATE TABLE " + tableName + " AS SELECT ROW(123) AS c1";
+        assertThatThrownBy(() -> assertUpdate(createTable))
+                .hasMessage(expectedMessage);
+
+        assertUpdate("CREATE TABLE " + tableName + " (c1 INTEGER) ");
+        String addColumn = "ALTER TABLE " + tableName + " ADD COLUMN c3 ROW(INTEGER)";
```

Comment:
Adding c3 column is weird when the table has only c1 column. 
```suggestion
        String addColumn = "ALTER TABLE " + tableName + " ADD COLUMN new_col ROW(INTEGER)";
```
Also, please inline the SQL.